From: Richard Cochran <richardcochran@gmail.com>
Date: Sun, 8 Apr 2012 14:38:10 +0000
Subject: e100: enable transmit time stamping.
Patch-mainline: v3.5-rc1
Git-commit: 48425b14929ef185377e3827ba7135d88343face
References: bnc#798929 FATE#313650

This patch enables software (and phy device) transmit time stamping.
Tested on an old PIII laptop with built in NIC.

Signed-off-by: Richard Cochran <richardcochran@gmail.com>
Tested-by: Aaron Brown <aaron.f.brown@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: Benjamin Poirier <bpoirier@suse.de>
---
 drivers/net/ethernet/intel/e100.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/intel/e100.c
+++ b/drivers/net/ethernet/intel/e100.c
@@ -1749,6 +1749,7 @@ static void e100_xmit_prepare(struct nic
 		skb->data, skb->len, PCI_DMA_TODEVICE));
 	/* check for mapping failure? */
 	cb->u.tcb.tbd.size = cpu_to_le16(skb->len);
+	skb_tx_timestamp(skb);
 }
 
 static netdev_tx_t e100_xmit_frame(struct sk_buff *skb,
